src/pages/benefits/viewAllBenefit/ApplicationDetails.tsx

```
@@ -120,6 +122,57 @@ const ApplicationDetails: React.FC = () => {
+    return allVerified;
+  };
+
+  // Helper function to check if eligibility has been evaluated
+  const checkEligibilityStatus = (criteria: any[]) => {
+    const hasEligibilityResults = criteria && criteria.length > 0;
+    setIsEligibilityChecked(hasEligibilityResults);
+    return hasEligibilityResults;
+  };
+
+  // Helper function to check if calculations are completed
+  const checkCalculationStatus = (amountData: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)